src/js/stores/account.js

```diff
@@ -60,8 +60,20 @@ export default Reflux.createStore({
   */
 
   onLogin(username, password, updatePayload) {
+    
     if (localStorage.getItem('webId')){
-      Account.login.completed(localStorage.getItem('webId')) 
+      
+      // Check if the cookie is still valid
+      fetch(`${proxy}/proxy?url=${localStorage.getItem('webId')}`, {
+        method: 'GET',
```

Comment:
Perhaps we could use something else but GET here? Get fetches the whole file and it's a bit costly if we want to only check if the cookie is active. Although at the moment the options request don't require a cookie, and we don't yet have implemented head requests.
So the ones that actually require a cookie are `PUT POST PATCH GET`.
You could perhaps send an empty `PATCH` request, this way you don't have to download the resource, but the code will still let you know if there's no cookie supplied.